app/services/profiles/update.rb

```diff
@@ -112,15 +113,16 @@ def update_profile
     end
 
     def conditionally_resave_articles
-      return unless core_profile_details_changed? && !@user.suspended?
+      return unless resave_articles? && !@user.suspended?
 
       Users::ResaveArticlesWorker.perform_async(@user.id)
     end
 
-    def core_profile_details_changed?
+    def resave_articles?
```

Comment:
I changed this method name because we're not only dealing with profile details in this method.